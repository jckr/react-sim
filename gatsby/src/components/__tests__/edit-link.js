import React from 'react';
import renderer from 'react-test-renderer';
import EditLink from '../edit-link';
describe('EditLink', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<EditLink />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
