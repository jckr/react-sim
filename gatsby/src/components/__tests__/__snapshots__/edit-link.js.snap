// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditLink renders correctly 1`] = `null`;
